Yoast/wordpress-seo
# Snippet preview title and description progress measurement overhaul

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* See https://github.com/Yoast/yoast-components/pull/562

## Test instructions

This PR can be tested by following these steps:
* See https://github.com/Yoast/yoast-components/pull/562

## Quality assurance


Fixes Yoast/wordpress-seo#9777
